src/sentry_core.c

```diff
@@ -1257,21 +1255,23 @@ sentry_clear_crashed_last_run(void)
     return success ? 0 : 1;
 }
 
-void
+sentry_uuid_t
 sentry_capture_minidump(const char *path)
 {
-    sentry_capture_minidump_n(path, sentry__guarded_strlen(path));
+    return sentry_capture_minidump_n(path, sentry__guarded_strlen(path));
 }
 
-void
+sentry_uuid_t
 sentry_capture_minidump_n(const char *path, size_t path_len)
 {
+    sentry_uuid_t eventId = sentry_uuid_nil();
```

Comment:
we use snake-case throughout the code-base.
```suggestion
    sentry_uuid_t event_id = sentry_uuid_nil();
```